wordpress-mobile/WordPress-iOS
# Quick Start: Two button alert style and skipped suggestion tracking

Refs #9447

This PR adds the notice style with two buttons. It also adds some functionality that makes that second button relevant, such as tracking the number of times a suggested Quick Start "tour" has been skipped, and hiding the checklist when Quick Start was dismissed.

![qs-two-button-notice](https://user-images.githubusercontent.com/517257/46046905-ad222d80-c0d7-11e8-8352-1e7f05bb9ec6.gif)


### Testing

Quite a few edge cases covered in this one. I think the script below follows them all, but if you find you want to repeatedly trigger the initial quick start fancy alert let me know, I have a terribly hacked snippet to do so. Also let me know if you want more mockups to reference than those in #9447 - 

To test:
- create a new site
- when Quick Start is offered tap "Yes"
- [ ] ensure the Quick Start menu is available
- tap "Switch Site" to return to the blog list
- return to the same site's details screen
- [ ] ensure the "View Site" tour is suggested
- [ ] ensure the suggestion notice matches mockups
- without responding to the suggestion notice, tap "Switch Site"
- [ ] ensure the notice is dismissed when backing out of the blog details screen
- tap "Not Now"
- repeat the switch site, return, then "not now" steps twice more
- [ ] ensure suggestions stop after being dismissed three times

- create a new site
- when QS is offered, tap "Not Now"
- [ ] ensure the Quick Start menu isn't there

- [ ] also ensure the existing notices still display correctly, such as when an async post completes
